### DISCUSSION TITLE
Can I generate C-arm image??

### DISCUSSION THREAD

**Discussion by Dev-MerB** [CERN/TIGRE#507] [Q&A] [answered]
Hello,

I want to apply TIGRE to generate a C-arm image using a CT volume, similar to the images below. 
![image](https://github.com/CERN/TIGRE/assets/39505929/ac0fd71b-f817-4c94-92c0-306cff069bd1)

But I only got this kind of image.(The background is black, and the entire area is visible)
- x-ray source mode : cone
- filter : none

![image](https://github.com/CERN/TIGRE/assets/39505929/0f2ee033-92f7-43d1-a9b1-4ed61341e83c)

Is there any sample code or documentation?

**Comment by AnderBiguri** [comment, marked as answer]
Hi @Dev-MerB 

The black->air images are X-ray absorption images, the ones needed to reconstruct. These are obtained by applying the Beer-Lambert law that dominates X-ray transmission, which is what a X-ray detector measures. The relationship is as follows:

`x_absorbtion=-log(x_transmission/max_white_value)`

Reconstruction happens in absorption because its linear, while transmission is not. You can change from one to the other by using this formula, given a known (or user defined) maximum white value in the detector (hardware/calibration specific). 

About the are that is visible: you have defined this. The structure `geo` defines what are the sizes/shapes of each part of the image, you need to change the values to the ones that you want to simulate. 

Hope that helps
